src/tests/ftest/osa/online_reintegration.py

```diff
@@ -8,8 +8,7 @@
 import random
 import threading
 
-from test_utils_pool import TestPool
-from test_utils_base import LabelGenerator
+from test_utils_pool import add_pool
```

Comment:
(pylint-syntax-error) Cannot import 'test_utils_pool' due to syntax error 'invalid syntax (<unknown>, line 53)'